cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/Storage.java

```diff
@@ -177,8 +178,8 @@ public StorageDiffResult diff(Storage other) {
      */
     private boolean compareSize(Quantity other) {
 
-        return Objects.isNull(this.size) ?
-                Objects.isNull(other) : this.size.getAmount().equals(other.getAmount());
+        return this.size == null ?
+                other == null : this.size.getAmount().equals(other.getAmount());
```

Comment:
I'm fine with using "== null" instead of `Objects.isNull` which is useful in Java 8 lambda filtering. What's your opinion @tombentley ?

Reply:
Yeah, `Objects.isNull` has been added purely for use as a method reference, we should use `== null` in non-functional code.

Reply:
Fine with me.